charts/consul/templates/ingress-gateways-deployment.yaml

```diff
@@ -55,6 +55,17 @@ spec:
         component: ingress-gateway
         ingress-gateway-name: {{ template "consul.fullname" $root }}-{{ .name }}
       annotations:
+        {{- if (and $root.Values.global.secretsBackend.vault.enabled $root.Values.global.tls.enabled) }}
+        "vault.hashicorp.com/agent-init-first": "true"
+        "vault.hashicorp.com/agent-inject": "true"
+        "vault.hashicorp.com/role": {{ $root.Values.global.secretsBackend.vault.consulCARole }}
```

Comment:
I'm always going to read this as someone yelling the first part of the name "Carol". "CAR-ole!"

Reply:
hahaha